packages/react-core/src/components/TextInput/examples/TextInput.md

```diff
@@ -26,7 +26,7 @@ class SimpleTextInput extends React.Component {
     const { value } = this.state;
 
     return (
-      <TextInput value={value} type="text" onChange={this.handleTextInputChange} aria-label="text input example" />
+      <TextInput isLeftTruncated value={value} type="text" onChange={this.handleTextInputChange} aria-label="text input example" />
```

Comment:
Clicking outside of the input field has different behaviors in different browsers. In chrome and safari, when you click outside the input field it returns to the start of the string that has been entered and truncates the right. In Firefox, it seems to stay at the end. Should this functionality be overwritten?